circuits-circom/hdfc_registration.circom

```diff
@@ -0,0 +1,86 @@
+pragma circom 2.1.5;
+
+include "circomlib/circuits/poseidon.circom";
+// include "./utils/email_verifier.circom";
+include "@zk-email/circuits/helpers/extract.circom";
+include "./stubs/email-verifier.circom";
+include "./utils/ceil.circom";
+include "./utils/extract.circom";
+include "./helpers/hdfc_helpers.circom";
+include "./regexes/from_regex.circom";
+include "./regexes/hdfc/hdfc_upi_subject.circom";
+
+
+template HdfcRegistrationEmail(max_header_bytes, max_body_bytes, n, k, pack_size) {
+    assert(n * k > 1024); // constraints for 1024 bit RSA
+
+    // Rounded to the nearest multiple of pack_size for extra room in case of change of constants
+    var max_email_from_len = ceil(21, pack_size); // RFC 2821: requires length to be 254, but we can limit to 21 (alerts@hdfcbank.net)
+    var max_email_to_len = ceil(35, pack_size); // TODO: Change this.
```

Comment:
for this, think we need to use RFC2821? So support max length of 254

Reply:
Do we want to? Is there a safe max email length that we can assume?

Reply:
Set it to 56 which is a safe length. Source: https://atdata.com/long-email-addresses/